markdown/dev/howtos/git/editing-on-github/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Editing freesewing content on Github
+---
+
+## Introduction
+This 'How to' is designed to help get you started editing content using the [Github website](https://github.com/freesewing).
+
+Github is a website that provides storage for all the Freesewing code and content.  It allows our supporters to make suggested edits and helps the administrators to review and implement approved changes. 
+
+If you are new to Github, you will need to [create an account](https://github.com/signup?user_email=&source=form-home-signup).Github For info, it is also possible to make changes using the Git desktop application and/or your computer's command line, but these advanced topics are beyond the scope of this 'How to'.
+
+All our pattern instructions and web pages (including this 'How to') are written in *Markdown*, which is an easy way to produce well-formated pages. Please see our [Markdown guide](https://freesewing.dev/guides/markdown) for more information on this.
+
+All the Markdown and images are stored in the Freesewing repository.  For security, most contributors don't have permission to change the code ourselves.  Instead we must follow three steps to make changes:
+
+1. *Fork* (create a copy of) the repository.  This will be your own copy and you can make all the changes you want.  There is no risk of you breaking the website or its content.
+2. Make the edits.
+3. Save your edits and submit a *pull request* to let the admins know about your changes.  The pull request is a way of saying *"Hey, I made some changes. I think you might like them and consider them for inclusion into your (original) repository."*
+
+The Github documentation provides a more [detailed explanation](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this process.  The steps here should be enough to get you started.
+
+## Fork the repository
+- On Github.com, navigate to the freesewing [repository](https://github.com/freesewing/freesewing)
+- In the top-right corner of the page, click **Fork**.  
+![Fork button](fork_button.png) You should see a page similar to this:
+![Create a fork dialogue](create-a-fork.png width="500")
+- Click the green "Create fork" button at the bottom of the page.
+
+
+## Make the edits
+- Navigate to the folder containing the file that you want to edit.
+
+To help you understand how the code is organised, here is the path to the Simone Instructions: <code>freesewing/markdown/org/docs/patterns/simone/instructions</code> This folder contains several files, one for each language. Please only update the English language `en.md` files. We have translation software to create other languages.
```

Comment:
I don't think that we use translation software? Perhaps I am wrong? As I understand it, instead "We have a translation team who will update the other language files."

Reply:
Yes, we use translation software: https://freesewing.dev/guides/translation

Reply:
To avoid all confusion:
- Yes we do use translation software: https://translate.freesewing.org/
- The non-English language files are automatically created/updated by this software. Any local changes will be overwritten

Which is why you should only ever update the `en.md` files.